test/integration/benchmark_test.go

```diff
@@ -25,10 +24,11 @@ import (
 	_ "github.com/aquasecurity/fanal/analyzer/library/yarn"
 	_ "github.com/aquasecurity/fanal/analyzer/os/alpine"
 	_ "github.com/aquasecurity/fanal/analyzer/os/amazonlinux"
-	_ "github.com/aquasecurity/fanal/analyzer/os/debianbase"
+	_ "github.com/aquasecurity/fanal/analyzer/os/debian"
```

Comment:
Have you renamed `debianbase` to `debian`?

Reply:
I split `debianbase` into `debian` and `ubuntu`.